.github/workflows/build-test.yml

```diff
@@ -41,7 +41,11 @@ jobs:
           java-version: '8'
           distribution: 'temurin'
       - name: Build with Maven
-        run: mvn clean install -pl -:gremlin-javascript -Dci --batch-mode -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
+        run: mvn clean install -pl -:gremlin-javascript,-:gremlin-dotnet-source,-:gremlin-dotnet-tests,-:gremlin-go,-:gremlin-python -Dci --batch-mode -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -Dcoverage
+      - name: Upload to Codecov
```

Comment:
Is there a reason/difference that this is part of the jdk8 run and not jdk11?

Reply:
I arbitrarily picked, but considering most of the Workflows are on JDK 11 perhaps its better to run it on that instead

Reply:
I picked it arbitrarily, perhaps its better to run coverage on JDK 11 since most workflows run it

Reply:
switched in [e7e7c9c](https://github.com/Bit-Quill/tinkerpop/pull/21/commits/e7e7c9c4e6ec0ae6b323f4ead4ae6de97f68b498)